src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -372,9 +371,9 @@ export default {
   },
 
   computed: {
-    ...mapGetters(['getEntityIncNo', 'isBComp', 'isRoleStaff', 'nrNumber']),
```

Comment:
This was the only getter that just returned the state value, so I made things consistent by using the state value and deleting the getter.